module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -366,6 +371,7 @@ namespace module::platform::darwin {
                     sensors->gyroscope = previousSensors->gyroscope;
                 }
                 else {
+                    // This smells wrong. y x -z? at the very least this needs an explanatory comment
                     sensors->gyroscope = Eigen::Vector3d(input.gyroscope.y, input.gyroscope.x, -input.gyroscope.z);
```

Comment:
Howevet I believe it should be like this?
sensors->gyroscope = Eigen::Vector3d(-input.gyroscope.y, -input.gyroscope.x, -input.gyroscope.z);